src/inference/src/cnn_network_ngraph_impl.cpp

```diff
@@ -589,14 +584,9 @@ StatusCode CNNNetworkNGraphImpl::serialize(const std::string& xmlPath,
 StatusCode CNNNetworkNGraphImpl::serialize(std::ostream& xmlBuf, std::ostream& binBuf, ResponseDesc* resp) const
     noexcept {
     try {
-        std::map<std::string, ngraph::OpSet> custom_opsets;
-        for (const auto& extension : _ie_extensions) {
-            auto opset = extension->getOpSets();
-            custom_opsets.insert(begin(opset), end(opset));
-        }
         ngraph::pass::Manager manager;
         using namespace ov::pass;
-        REGISTER_PASS(manager, Serialize, xmlBuf, binBuf, custom_opsets, ov::pass::Serialize::Version::IR_V10)
+        REGISTER_PASS(manager, Serialize, xmlBuf, binBuf, ov::pass::Serialize::Version::IR_V10)
```

Comment:
looks like we still use extensions here. 
I suppose a simple removal can break user's code?

Reply:
Looks like we will be able to remove this code only together with IE API

Reply:
@ilyachur & @ilya-lavrenov So for this interfaces, we need to delay to next release?

Reply:
if one public API depends on another, we need to remove dependency first, then dependent API